SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -294,6 +294,11 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
 	UPROPERTY(EditAnywhere, config, Category = "Debug", meta = (MetaClass = "SpatialDebugger"))
 	TSubclassOf<ASpatialDebugger> SpatialDebugger;
 
+	/** Toggle to allow debug checking of invalid data modification on non-auth workers */
+	UPROPERTY(EditAnywhere, config, Category = "Debug")
+	bool bEnableSpatialDataDebugger;
```

Comment:
"bEnableSpatialDataDebugger" seems a bit too generic a name?

Reply:
I can't think of a better name but I'm happy to change it if someone has a suggestion?

Reply:
Changed to bSpatialAuthorityDebugger